packages/view-types/genomic-profiles/src/HiGlassLazy.js

```diff
@@ -1,33 +1,15 @@
 import React, {
   useMemo, useEffect, useRef, Suspense, useState,
 } from 'react';
-import ReactDOM from 'react-dom';
-import dynamicImportPolyfill from 'dynamic-import-polyfill';
 import register from 'higlass-register';
 import { ZarrMultivecDataFetcher } from 'higlass-zarr-datafetchers';
-import { asEsModule } from '@vitessce/utils';
 import { useGridItemSize, useCoordination } from '@vitessce/vit-s';
 import { COMPONENT_COORDINATION_TYPES } from '@vitessce/constants-internal';
-import { createWarningComponent } from './utils';
 import { useStyles } from './styles';
 
-// TODO(monorepo)
-// import packageJson from '../../../package.json';
-// const PIXI_BUNDLE_VERSION = packageJson.dependencies['window-pixi'];
-// const HIGLASS_BUNDLE_VERSION = packageJson.dependencies.higlass;
-// const BUNDLE_FILE_EXT = process.env.NODE_ENV === 'development' ? 'js' : 'min.js';
-const PIXI_BUNDLE_VERSION = '5.3.3';
-const HIGLASS_BUNDLE_VERSION = '1.11.4';
-const BUNDLE_FILE_EXT = true ? 'js' : 'min.js';
-const PIXI_BUNDLE_URL = `https://unpkg.com/window-pixi@${PIXI_BUNDLE_VERSION}/dist/pixi.${BUNDLE_FILE_EXT}`;
-const HIGLASS_BUNDLE_URL = `https://unpkg.com/higlass@${HIGLASS_BUNDLE_VERSION}/dist/hglib.${BUNDLE_FILE_EXT}`;
+const HIGLASS_BUNDLE_VERSION = '1.11.11';
 const HIGLASS_CSS_URL = `https://unpkg.com/higlass@${HIGLASS_BUNDLE_VERSION}/dist/hglib.css`;
```

Comment:
nit, but this doesn't work now (or at least is incorrect) since we are on the fork?  also could you post a link to the fork?  even if it's yours, it doesn't seem the changes are committed in it so just saying that would be a good bread crumb trail here.

Reply:
And just updated the CSS url